manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: StatsifyInc.StatsifyOverlay
+PackageVersion: 2.7.2
+PackageLocale: en-US
+Publisher: Statsify Inc
+PackageName: Statsify Overlay

```

Comment:
```suggestion
PackageName: statsify-overlay
```
![image](https://user-images.githubusercontent.com/11600822/120932950-cf293600-c6ef-11eb-84a5-406db06bc739.png)

Other people may want to upgrade the application via the `winget upgrade` command; so the data from the Control Panel must match the one shown in the manifest.